iron-ajax.html

```diff
@@ -452,6 +456,13 @@
         options: requestOptions
       }, {bubbles: this.bubbles});
 
+      if(this.bubbles){
```

Comment:
Spoke with @rictic and we believe that this event firing is fairly inexpensive and that it would be more-consistent to fire these events regardless of `this.bubbles`.

Recommend firing these `iron-ajax-<event>` events in the same fashion with `bubbles: this.bubbles`.